package.json

```diff
@@ -89,6 +89,7 @@
     "istanbul": "^0.4.4",
     "istanbul-instrumenter-loader": "^3.0.0",
     "jimp": "^0.2.28",
+    "jsdom": "^13.1.0",
```

Comment:
Is it possible to use cheerio for this instead of jsdom? I suspect it will be much faster/lighter.

Reply:
Andy tested cheerio and the difference was far smaller than I expected - about 4 seconds across all files. So it probably doesn't matter which one we use. Andy also pointed out that jsdom also executes scripts which might be useful if we ever end up adding the class dynamically (e.g. due to using templates).